notebooks/cookbook/Querying and Modifying Charges and Other Parameters.ipynb

```diff
@@ -25,6 +25,28 @@
     "system = forcefield.createSystem(pdb.topology)"
    ]
   },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "6d26c481",
+   "metadata": {},
+   "source": [
+    "Let's also create a `Simulation` with this system."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "id": "552481f3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "integrator = LangevinMiddleIntegrator(\n",
+    "    300 * unit.kelvin, 1 / unit.picosecond, 0.004 * unit.picoseconds)\n",
+    "simulation = Simulation(pdb.topology, system, integrator)\n",
+    "simulation.context.setPositions(pdb.positions)"
```

Comment:
Why did you add this?  It's extra code that isn't relevant to subject.

Reply:
I thought you would want to show how to modify the `Context`?

Reply:
Should we just mention it in the text then?